layer/executables/entrypoint/common.py

```diff
@@ -16,17 +20,34 @@
 ENV_LAYER_RUN_ID = "LAYER_RUN_ID"
 
 RunnerFunction = Callable[[FunctionDefinition], Any]
-RunFunction = Callable[[FunctionDefinition, Config, Fabric, str], Any]
+RunFunction = Callable[
+    [FunctionDefinition, LayerClient, RunProgressTracker, Fabric, str], Any
+]
+
+logger = logging.getLogger(__name__)
 
 
 def make_runner(run_function: RunFunction) -> RunnerFunction:
     def runner(definition: FunctionDefinition) -> Any:
         def inner() -> Any:
             _initialize(definition)
             config: Config = asyncio_run_in_thread(ConfigManager().refresh())
-            return run_function(
-                definition, config, _get_display_fabric(), _get_run_id()
+            progress_tracker = get_progress_tracker(
+                url=config.url,
+                project_name=definition.project_name,
+                account_name=definition.account_name,
```

Comment:
we could start passing `ProjectFullName` in places which require both `account_name` and `project_name`

Reply:
agreed, noting for a subsequent refactor PR